tests/Unit/Domain/Test_Domain.cpp

```diff
@@ -97,8 +102,13 @@ void test_1d_domains() {
             std::make_unique<CoordinateMap<Frame::BlockLogical, Frame::Inertial,
                                            CoordinateMaps::Affine>>(
                 make_coordinate_map<Frame::BlockLogical, Frame::Inertial>(
-                    CoordinateMaps::Affine{-1., 1., 2., 0.}))));
+                    CoordinateMaps::Affine{-1., 1., 2., 0.}))),
+        {}, {"Left", "Right"}, {{"All", {"Left", "Right"}}});
     CHECK_FALSE(domain_no_corners.is_time_dependent());
```

Comment:
can you add a test that checks that if the block groups are different the domain is not equal? I find the current `==` behavior unintuitive